presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -510,6 +510,17 @@ private static LongWriteFunction timestampWriteFunction(TimestampType timestampT
         };
     }
 
+    @Override
+    public void setColumnComment(JdbcIdentity identity, JdbcTableHandle handle, JdbcColumnHandle column, Optional<String> comment)
+    {
+        String sql = format(
+                "COMMENT ON COLUMN %s.%s IS '%s'",
+                quoted(handle.getRemoteTableName()),
+                quoted(column.getColumnName()),
+                comment.orElse(""));
```

Comment:
`orElse(null)`?

Reply:
I think generated sql from `orElse(null)` will be `IS 'null'`. My initial local implementation was like this. What do you think about it?
```java
        String sql = format(
                "COMMENT ON COLUMN %s.%s IS %s",
                quoted(handle.getRemoteTableName()),
                quoted(column.getColumnName()),
                comment.isPresent() ? format("'%s'", comment.get()) : "NULL");
```

Reply:
we should not add a four-letter comment (with letters n, u, l, l) unless user explicitly requested that

you can think of that `String.valueOf(null)` returns `"null"` (and not something else)  is Presto's impl details and should not affected user visible behavior.